### PR TITLE
Razor VSCode 5.0 Preview 3

### PR DESCRIPTION
**Pull request by TanayParikh**
* Razor support for `<text>` tag completions.
* Ability to restart the Razor Language Server to activate changes to the `razor.trace` level.
* Bug fixes and performance improvements.

Covers: https://github.com/dotnet/aspnetcore/milestone/97?closed=1

@NTaylorMullen @ryanbrandenburg @ajaybhargavb Please look over the changelog / test-plan to ensure I didn't miss anything or if I can word something better.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +11,9 @@
 ## 1.21.16 (Not yet released)
 * Support for .NET Core 3.1 in csx files (PR: [#1731](https://github.com/OmniSharp/omnisharp-roslyn/pull/1731))
 * Update the minimal MSBuild to better support .NET 5 Previews ([omnisharp-vscode#3653](https://github.com/OmniSharp/omnisharp-vscode/issues/3653), PR: [#1746](https://github.com/OmniSharp/omnisharp-roslyn/pull/1746))
+* Razor support for `<text>` tag completions.
+* Ability to restart the Razor Language Server to activate changes to the `razor.trace` level.
+* Bug fixes and performance improvements.
```
Comment (JoeRobich) [inline CHANGELOG.md]: We have a pre-release build of 1.21.16 for .NET 5 Preview 2 validation which will likely be released on the 30th. 

```suggestion
## 1.21.17 (Not yet released)
* Updated Razor support (PR:[#3696](https://github.com/OmniSharp/omnisharp-vscode/pull/3696))
  * Razor support for `<text>` tag completions.
  * Ability to restart the Razor Language Server to activate changes to the `razor.trace` level.
  * Bug fixes and performance improvements.

## 1.21.16 (March 30, 2020)
* Support for .NET Core 3.1 in csx files (PR: [#1731](https://github.com/OmniSharp/omnisharp-roslyn/pull/1731))
* Update the minimal MSBuild to better support .NET 5 Previews ([omnisharp-vscode#3653](https://github.com/OmniSharp/omnisharp-vscode/issues/3653), PR: [#1746](https://github.com/OmniSharp/omnisharp-roslyn/pull/1746))
```

**Comment by TanayParikh** [comment]
Travis-CI check still shows in progress on the PR, but seems to have gone through already on the Travis-CI end:

https://travis-ci.org/github/OmniSharp/omnisharp-vscode/jobs/667905947?utm_medium=notification&utm_source=github_status

**Comment by TanayParikh** [comment]
Travis-CI check seems to be passing but the github check isn't getting updated:
https://travis-ci.org/github/OmniSharp/omnisharp-vscode/builds/668850141?utm_medium=notification&utm_source=github_status

Could this be manually pushed through?

**Comment by NTaylorMullen** [comment]
Hmm @JoeRobich the Travis build is succeeding but it doesn't look to be reporting to GitHub properly.

**Pull request merged**